scripts/jenkins-yoctobuild-build.sh

```diff
@@ -276,7 +276,8 @@ then
         # use original path when building qemu
         export PATH=$OLD_PATH
         build_custom_qemu
-        prepare_and_set_PATH
+        ( cd $BUILDDIR && prepare_and_set_PATH )
```

Comment:
I think you should rather change directory back inside `build_custom_qemu` (for example using `pushd` and `popd`). Otherwise the directory will still be wrong after this call.